src/transformers/models/segformer/modeling_tf_segformer.py

```diff
@@ -201,9 +201,9 @@ def __init__(self, config: SegformerConfig, hidden_size: int, **kwargs):
         self.dense = tf.keras.layers.Dense(hidden_size, name="dense")
         self.dropout = tf.keras.layers.Dropout(config.hidden_dropout_prob)
 
-    def call(self, hidden_states: tf.Tensor) -> tf.Tensor:
+    def call(self, hidden_states: tf.Tensor, training: bool = False) -> tf.Tensor:
         hidden_states = self.dense(hidden_states)
-        hidden_states = self.dropout(hidden_states)
+        hidden_states = self.dropout(hidden_states, training=training)
```

Comment:
We still pass this argument, right?

Reply:
We can avoid it since these are supposed to be set automatically during training. But I find that explicitly specifying it gives me mental peace.

Reply:
Thank you for the information!